packages/e2e-tests/helpers/source-panel.ts

```diff
@@ -28,6 +28,7 @@ export async function addBreakpoint(
   if (url) {
     await openSourceExplorerPanel(page);
     await openSource(page, url);
+    await page.waitForSelector('[data-test-name="Source"]:visible');
```

Comment:
Maybe this line would be better placed inside of the `openSource` helper function?